packages/cli/test/unit/commands/access/remove.unit.test.ts

```diff
@@ -0,0 +1,25 @@
+import {stdout, stderr} from 'stdout-stderr'
+import * as nock from 'nock'
+import {expect} from 'chai'
+import Cmd  from '../../../../src/commands/access/remove'
+import runCommand from '../../../helpers/runCommand'
+import {collaboratorsPersonalApp} from '../../../helpers/stubs/delete'
+let apiDelete: { done: () => any }
+describe('heroku access:remove', () => {
+  context('with either a personal or org app', () => {
+    beforeEach(() => {
+      apiDelete = collaboratorsPersonalApp('myapp', 'raulb@heroku.com')
+    })
+    afterEach(() => nock.cleanAll())
+    it('removes the user from an app', () => {
+      return runCommand(Cmd, [
+        '--app',
+        'myapp',
+        'raulb@heroku.com',
+      ])
+        .then(() => expect('').to.eq(stdout.output))
+        .then(() => expect('Removing raulb@heroku.com access from the app myapp...\nRemoving raulb@heroku.com access from the app myapp... done\n').to.eq(stderr.output))
+        .then(() => apiDelete.done())
```

Comment:
I was always changing this to use `async/await` notation. Is it necessary to do it or is it just a linting suggestion that can be ignored?

Reply:
I have been ignoring it. I started out changing them to async/await, but decided to stop in the interest of time.